style.css

```diff
@@ -102,3 +136,276 @@ span {
   color: #505f79;
   width: 1.2rem;
 }
+
+.cards {
+  display: grid;
+  height: auto;
+  gap: 2rem;
+  margin-left: 1rem;
+  width: 90%;
+}
+
+.card1,
+.card2,
+.card3,
+.card4 {
+  display: flex;
+  flex-direction: column;
+  border: 0.063rem solid #dfe1e6;
+  border-radius: 1rem;
+  background: #fff;
+  height: auto;
+  padding: 1rem;
+  width: 90%;
+}
+
+.mobileimage1 {
+  background-image: url('/images/mobileImg1.jpg');
+  background-repeat: no-repeat;
+  height: 13.75rem;
+  width: 100%;
+  border-radius: 0.5rem;
+}
```

Comment:
- [ ] Good job so far in implementing the changes requested by the previous reviewer, however, you may have created errors in making those changes, for example:  you forgot to put a dot `.` in front of all your image paths and thus, none of your images is showing correctly. it should be `url('./images/mobileImg1.jpg');` instead of this `url('/images/mobileImg1.jpg');` , Please resolve it in all your image paths.
- [ ] As mentioned by the previous reviewers, I can see that your design is not still matching the given design, matching your design to the given design requirements enhances the chances of creating a successful and well-rounded design solution that meets the needs of users, complies with regulations, and performs effectively and efficiently.  Please ensure that you pay close attention this design [template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=0-1&mode=design&t=aPAqREn7zR7JAx5t-0):slightly_smiling_face: 

The example below shows the difference between yours and the accepted one: :arrow_down: 

 <table>
    <tr>
      <td>Yours </td>
      <td>Accepted type</td>
    </tr>
    <tr>
      <td>

 ![Student Design](https://github.com/efmomoh/Portfolio-setup-and-mobile-first/assets/43528594/541d19ee-3561-4235-af19-7a1bcbdc32c8) 

</td>
      <td>

![Figma Design](https://github.com/efmomoh/Portfolio-setup-and-mobile-first/assets/43528594/c6d5d961-7066-42aa-b245-3be7a2ada4a9)

</td>
    </tr>
  </table>